.github/workflows/create-release.yml

```
@@ -175,12 +175,20 @@ jobs:
         cp install_sct.bat install_sct-${{ github.event.inputs.milestone_title }}_win.bat
         perl -pi -e 's/set git_ref=master/set git_ref=${{ github.event.inputs.milestone_title }}/' install_sct-${{ github.event.inputs.milestone_title }}_win.bat
 
+    - name: Create ${{ github.event.inputs.milestone_title }} Linux/macOS install scripts
+      # NB: `install_sct` works for both Linux and macOS platforms. So, here, we simply duplicate the script for both
+      # platforms (which will allow us to collect some rudimentary statistics + make our installation steps clearer).
+      run: |
+        cp install_sct install_sct-${{ github.event.inputs.milestone_title }}_linux.sh
+        perl -pi -e 's/SCT_GIT_REF="master"/SCT_GIT_REF="${{ github.event.inputs.milestone_title }}"/' install_sct-${{ github.event.inputs.milestone_title }}_linux.sh
+        cp install_sct-${{ github.event.inputs.milestone_title }}_linux.sh install_sct-${{ github.event.inputs.milestone_title }}_macos.sh
+
     - uses: ncipollo/release-action@v1
       name: Create release
       id: create_release
       with:
         tag: ${{ github.event.inputs.milestone_title }}
         token: ${{ secrets.GITHUB_TOKEN }}
         bodyFile: ".github/workflows/release-body.md"
-        artifacts: "install_sct-${{ github.event.inputs.milestone_title }}_win.bat"
+        artifacts: "install_sct-${{ github.event.inputs.milestone_title }}_win.bat,install_sct-${{ github.event.inputs.milestone_title }}_linux.sh,install_sct-${{ github.event.inputs.milestone_title }}_macos.sh"
```

By my reading of the https://yaml-multiline.info/ site, maybe this would work? It would be a flow scalar, double-quoted, with newlines escaped so that they don't turn into spaces.
```suggestion
        artifacts: "install_sct-${{ github.event.inputs.milestone_title }}_win.bat,\
                    install_sct-${{ github.event.inputs.milestone_title }}_linux.sh,\
                    install_sct-${{ github.event.inputs.milestone_title }}_macos.sh"
```
Looking at [the source of release-action](https://github.com/ncipollo/release-action/blob/d51f1872a4078bf85434f4b20c9bdbc4eb02f295/src/ArtifactGlobber.ts#L20-L22), I think the allowed syntax is that the list is split on both commas and newlines, and that leading whitespace is then trimmed for each path. But I'm not sure. I admit I'm a bit puzzled by the strange warnings you got.

---

Delightful! Thank you for the suggestion. :hearts: